.pipelines/singletenancy/azure-cni-overlay/azure-cni-overlay-e2e-step-template.yaml

```diff
@@ -132,3 +107,40 @@ steps:
       name: "WindowsV4OverlayDatapathTests"
       displayName: "Windows v4Overlay Datapath Tests"
       retryCountOnTaskFailure: 3
+
+    - task: AzureCLI@1
+      inputs:
+        azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+        scriptLocation: "inlineScript"
+        scriptType: "bash"
+        addSpnToEnvironment: true
+        inlineScript: |
+          set -e
+          kubectl get po -owide -A
+          clusterName=${{ parameters.clusterName }}
+          echo "Restarting nodes"
+          for val in $(az vmss list -g MC_${clusterName}_${clusterName}_$(REGION_AKS_CLUSTER_TEST) --query "[].name" -o tsv); do
+            make -C ./hack/aks restart-vmss AZCLI=az CLUSTER=${clusterName} REGION=$(REGION_AKS_CLUSTER_TEST) VMSS_NAME=${val}
+          done
+      displayName: "Restart Nodes"
+
+    - task: AzureCLI@1
+      inputs:
+        azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+        scriptLocation: "inlineScript"
+        scriptType: "bash"
+        addSpnToEnvironment: true
+        inlineScript: |
+          cd test/integration/load
+          clusterName=${{ parameters.clusterName }}
+          make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${clusterName}
```

Comment:
I wonder why this doesn't cause a test failure in the pipelines either?

Reply:
Good catch. Probably because we got the kubeconfig in an earlier step. Thus it was not needed in the first place. Also, since make goes into another shell, unless you capture the error codes with `set -e or -o pipefail` it will not fail the pipeline. 

Will fix this.